assets/js/common.js

```diff
@@ -152,7 +152,12 @@ var hivepress = {
 
 						return template;
 					},
-				};
+				},
+				fieldOptions = field.data('options');
+
+			if (fieldOptions && fieldOptions.closeOnSelect) {
```

Comment:
I noticed you also use disableSearch in data-options but this code contains specific solution for closeOnSelect with explicit check, I don't see where disableSearch is passed, does it work?

You can change this to using the $.extend function to just get the data-options array and override the defaults, this way we can support any options passed to Select2 via the HTML attribute.

Reply:
disableSearch is calling from bookings extension

Reply:
disableSearch in JS implemented in other task, related with time-slots and yes, it works